archive/walk.go

```diff
@@ -212,8 +212,8 @@ func (c Chunk) Range() string {
 	return fmt.Sprintf("[%d-%d]", c.First, c.Last)
 }
 
-func chunkTsLess(dir zbuf.Direction, iTs nano.Ts, iKid ksuid.KSUID, jTs nano.Ts, jKid ksuid.KSUID) bool {
-	if dir == zbuf.DirTimeForward {
+func chunkTsLess(dir zbuf.Order, iTs nano.Ts, iKid ksuid.KSUID, jTs nano.Ts, jKid ksuid.KSUID) bool {
```

Comment:
Change the name of the dir parameter to order in all of these functions.

```suggestion
func chunkTsLess(order zbuf.Order, iTs nano.Ts, iKid ksuid.KSUID, jTs nano.Ts, jKid ksuid.KSUID) bool {
```